### PR TITLE
update version

### PR DESCRIPTION
**Pull request by wurstbrot**

### CODE REVIEW

**Comment by christopher-cudennec** [comment]
Version 1.x is discontinued. Shouldn't that part be removed from the documentation completely @dhenneke ?

**Comment by Fox32** [comment]
We don't generate releases for the `releases/2.0.0` branch. It doesn't even exist.

**Comment by Fox32** [comment]
See https://github.com/SDA-SE/sda-dropwizard-commons/commit/1c454d525d1be00a9245ed9a3fd24915e0586ed6

File: RELEASES.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ This document describes the release process of SDA Dropwizard Commons.
 ## Happy Path
 
 The release process is automated.
-It generates new releases for all merges to `master` or `release/1.x.x` and uploads all releases to Maven Central.
+It generates new releases for all merges to `master` or `release/2.x.x` and uploads all releases to Maven Central.
```
Comment (Fox32) [inline RELEASES.md:8]:
```suggestion
It generates new releases for all merges to `master` and uploads all releases to Maven Central.
```

Please use semantic commits for the commit message

**Comment by wurstbrot** [comment]
@Fox32 so you are pointing out that it should be:
> It generates new releases for all merges to `master` and uploads all releases to Maven Central.

**Review by dhenneke — APPROVED** [review]
Thanks!

**Pull request merged**